src/compiler/crystal/config.cr

```diff
@@ -8,12 +8,16 @@ module Crystal
       version_and_sha.first
     end
 
+    def self.llvm_version
+      LibLLVM::VERSION
+    end
+
     def self.description
       version, sha = version_and_sha
       if sha
-        "Crystal #{version} [#{sha}] (#{date})"
+        "Crystal #{version} [#{sha}] (#{date}) LLVM #{llvm_version}"
       else
-        "Crystal #{version} (#{date})"
+        "Crystal #{version} (#{date}) LLVM #{llvm_version}"
```

Comment:
I think one line is fine for now. Maybe we can discuss the general output of `crystal version` in another issue :-)